Corrected setup for cosmic rune task

maxwelldixon opened the pull request:
Updated "Tiara recommended unless using essence pouches" to "Cosmic tiara or Catalytic tiara recommended if not using abyss." Resolves #1813. 

![Screenshot from 2024-10-21 12-56-57](https://github.com/user-attachments/assets/5e85ce72-6977-4a82-8dc8-6354adb64fe3)

Zoinkwiz reviewed (commented):
I'm not 100% sure, but would it make sense for the pouches to still be an option? I can't check currently, but I believe they're droppable, which means you can fill a pouch up, enter the altar with a talisman, drop the talisman and a few essence in the altar, drop the pouch, then pick up the essence so you have 28.

maxwelldixon commented:
@Zoinkwiz, you're correct. I just checked by entering via the cosmic altar with a filled small pouch (3), a cosmic talisman, and 26 rune essence. Then dropped the talisman, dropped 2 rune essence, emptied the small pouch (3), dropped the small pouch, and finally picked up 1 rune essence. Which filled by inventory with 28 rune essence and I crafted 56 cosmic rune simultaneously. Thank you! I didn't consider dropping 🤣
